strapi/strapi
# Api token v2/time selector

### What does it do?

- Adds a duration selector drop down
- Adds the frontend logic for the Duration selector of an API Token

### Why is it needed?

- it's needed to be able to choose an expiration time for an API token

### How to test it?

Run the application and check if it follows the following acceptance criteria:

- Given I’m a user who has access to the API tokens settings
  - when I access the creation view of a token

    - then I want to find a required selector to pick a duration time (following the design) with the default option of “Select” and the options of 7 days, 30 days, 90 days, and no limits

  - when I access the edit view of a token

    - then I want the duration time selector to be inactive and not selectable (uneditable)
